gravitee-repository-elasticsearch/src/main/java/io/gravitee/repository/elasticsearch/healthcheck/query/AbstractElasticsearchQueryCommand.java

```diff
@@ -77,4 +77,16 @@ protected String createQuery(final String templateName, final Query<T> query) {
 		
 		return request;
 	}
+
+	String createQuery(final String templateName, final Query<T> query, Long newFrom, Long newTo ) {
```

Comment:
Should be in ../analytics/query/AbstractElasticsearchQueryCommand.java

Reply:
when i made the review that did not compile... you changed createQuery method in ../**analytics**/query/DateHistogramQueryCommand.java while you change the signature in ../**healthcheck**/query/AbstractElasticsearchQueryCommand.java (and not in ./**analytics**/query/AbstractElasticsearchQueryCommand.java)